src/main/java/uk/gov/hmcts/reform/judicialbooking/v1/V1.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.reform.judicialbooking.v1;
+
+public final class V1 {
+
+    private V1() {
+    }
+
+    public static final class MediaType {
+
+        private MediaType() {
+        }
+
+        // External API
+        public static final String SERVICE = "application/vnd.uk.gov.hmcts.judicial-booking-service";
+        public static final String CREATE_BOOKING = SERVICE
+                + ".create-booking+json;charset=UTF-8;version=1.0";
+        public static final String GET_BOOKINGS = SERVICE
+                + ".get-bookings+json;charset=UTF-8;version=1.0";
+    }
+
+    public static final class Error {
+        private Error() {
+        }
+
+        public static final String INVALID_REQUEST = "Request is not valid as per validation rule";
+        public static final String INVALID_ROLE_NAME = "Invalid role name in the request";
+        public static final String BAD_REQUEST_INVALID_PARAMETER = "Invalid Parameter";
+        public static final String BAD_REQUEST_INVALID_DATETIME = "Invalid Datetime Parameter";
+        public static final String NO_RECORDS_FOUND_BY_ACTOR = "Role Assignment not found for Actor";
```

Comment:
It should be set to Booking not found for Actor

Reply:
we can remove all other not required constants

Reply:
I am not using these properties. I copied from Conor branch. I will clean this up.